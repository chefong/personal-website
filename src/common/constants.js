import hamburgerMenuAnimation from '~/assets/menu-animation.json';

export const gaEvents = {
    PAGE_RENDERED: 'page_rendered',
    ABOUT_NAV_CLICKED: 'about_nav_clicked',
    PROJECTS_NAV_CLICKED: 'projects_nav_clicked',
    EXPERIENCE_NAV_CLICKED: 'experience_nav_clicked',
    THEME_TOGGLE_CLICKED: 'theme_toggle_clicked',
    RESUME_CLICKED: 'resume_clicked',
    CONTACT_ME_CLICKED: 'contact_me_clicked',
    GITHUB_CLICKED: 'github_icon_clicked',
    LINKEDIN_CLICKED: 'linkedin_icon_clicked',
    SPOTIFY_CLICKED: 'spotify_icon_clicked',
    MEDIUM_CLICKED: 'medium_icon_clicked',
    MAIL_CLICKED: 'mail_icon_clicked',
};

export const navItems = [
    {
        to: 'about',
        num: '01.',
        name: 'About',
        clickEventId: gaEvents.ABOUT_NAV_CLICKED,
    },
    {
        to: 'projects',
        num: '02.',
        name: 'Projects',
        clickEventId: gaEvents.PROJECTS_NAV_CLICKED,
    },
    {
        to: 'experience',
        num: '03.',
        name: 'Experience',
        clickEventId: gaEvents.EXPERIENCE_NAV_CLICKED,
    },
];

export const projects = [
    {
        nameId: 'PROJECTS_PATCH',
        descriptionId: 'PROJECTS_PATCH_DESCRIPTION',
        githubLink: 'https://github.com/willshiao/Patch',
    },
    {
        nameId: 'PROJECTS_CHEATERBEATER',
        descriptionId: 'PROJECTS_CHEATERBEATER_DESCRIPTION',
        githubLink: 'https://github.com/willshiao/CheaterBeater',
    },
    {
        nameId: 'PROJECTS_BAUT',
        descriptionId: 'PROJECTS_BAUT_DESCRIPTION',
        githubLink: 'https://github.com/KirbyDownB/GEICOChatBot',
    },
    {
        nameId: 'PROJECTS_NUTSHELL',
        descriptionId: 'PROJECTS_NUTSHELL_DESCRIPTION',
        githubLink: 'https://github.com/chefong/nutshell',
    },
    {
        nameId: 'PROJECTS_STOCK_OVERFLOW',
        descriptionId: 'PROJECTS_STOCK_OVERFLOW_DESCRIPTION',
        githubLink: 'https://github.com/kimjihwan0208/stockoverflow',
    },
    {
        nameId: 'PROJECTS_JOYRIDE',
        descriptionId: 'PROJECTS_JOYRIDE_DESCRIPTION',
        githubLink: 'https://github.com/ericong18/joyride',
    },
];

export const experiences = [
    {
        imagePath: {
            light: 'imgs/experience/opentable.png',
            dark: 'imgs/experience/opentable-white.png',
        },
        nameId: 'EXPERIENCE_OPENTABLE',
        positionId: 'SOFTWARE_ENGINEER_1',
        duration: 'February 2021 - Present',
        teamId: 'EXPERIENCE_OPENTABLE_TEAM_2',
        initiativesId: 'EXPERIENCE_OPENTABLE_INITIATIVES_2',
        technologiesId: 'EXPERIENCE_OPENTABLE_TECHNOLOGIES_2',
    },
    {
        imagePath: {
            light: '/imgs/experience/intuit.svg',
            dark: '/imgs/experience/intuit-white.svg',
        },
        nameId: 'EXPERIENCE_INTUIT',
        positionId: 'SOFTWARE_ENGINEER_1',
        duration: 'July 2020 - February 2021',
        teamId: 'EXPERIENCE_INTUIT_TEAM',
        initiativesId: 'EXPERIENCE_INTUIT_INITIATIVES',
        technologiesId: 'EXPERIENCE_INTUIT_TECHNOLOGIES',
    },
    {
        imagePath: {
            light: 'imgs/experience/opentable.png',
            dark: 'imgs/experience/opentable-white.png',
        },
        nameId: 'EXPERIENCE_OPENTABLE',
        positionId: 'SOFTWARE_ENGINEERING_INTERN',
        duration: 'June 2019 - August 2019',
        teamId: 'EXPERIENCE_OPENTABLE_TEAM_1',
        initiativesId: 'EXPERIENCE_OPENTABLE_INITIATIVES_1',
        technologiesId: 'EXPERIENCE_OPENTABLE_TECHNOLOGIES_1',
    },
];

export const handles = [
    {
        name: 'GitHub',
        icon: ['fab', 'github'],
        link: 'https://github.com/chefong',
        clickEventId: gaEvents.GITHUB_CLICKED,
    },
    {
        name: 'LinkedIn',
        icon: ['fab', 'linkedin-in'],
        link: 'https://www.linkedin.com/in/ericong18/',
        clickEventId: gaEvents.LINKEDIN_CLICKED,
    },
    {
        name: 'Spotify',
        icon: ['fab', 'spotify'],
        link: 'https://open.spotify.com/user/ericong18?si=H0kHkFdxQ-GyKIlfLWzDqg',
        clickEventId: gaEvents.SPOTIFY_CLICKED,
    },
    {
        name: 'Medium',
        icon: ['fab', 'medium'],
        link: 'https://medium.com/@ericong18',
        clickEventId: gaEvents.MEDIUM_CLICKED,
    },
    {
        name: 'Email',
        icon: 'envelope',
        link: 'mailto:ericong18@gmail.com',
        clickEventId: gaEvents.MAIL_CLICKED,
    },
];

export const likes = [
    'ABOUT_LIKE_TEA',
    'ABOUT_LIKE_WATERMELON',
    'ABOUT_LIKE_MUSIC',
    'ABOUT_LIKE_DESIGN',
    'ABOUT_LIKE_PREMIER_LEAGUE',
];

export const dislikes = ['ABOUT_DISLIKE_CILANTRO'];

export const themeColors = {
    light: '#fffefb',
    dark: '#222222',
    tan: '#f1e7d0',
};

export const lottieProps = {
    loop: false,
    autoplay: false,
    animationData: hamburgerMenuAnimation,
};

export const reactScrollLinkProps = {
    smooth: true,
    offset: -150,
    duration: 750,
};

// Segment frames for menu open and close
export const menuOpenSegments = [30, 70];
export const menuCloseSegments = [100, 140];

// Box shadow styling for when the nav menu is open
export const menuBoxShadow = '0px 4px 28px rgba(0, 0, 0, 0.06)';

// Menu transition styling for menu
export const menuTransition = 'all 0.5s ease';

export const themeOrder = ['light', 'tan', 'dark'];

export const iconSize = '28px';

export const iconColors = {
    light: themeColors.dark,
    dark: themeColors.light,
    tan: themeColors.dark,
};

export const toastOptions = {
    style: {
        padding: '16px',
        fontFamily: 'Greycliff',
        backgroundColor: '#303030',
        color: 'white',
    },
};

export const particlesParams = {
    particles: {
        number: {
            value: 80,
            density: {
                enable: true,
                value_area: 1500,
            },
        },
        line_linked: {
            enable: true,
            opacity: 0.02,
        },
        move: {
            direction: 'right',
            speed: 0.05,
        },
        size: {
            value: 1.25,
        },
        opacity: {
            anim: {
                enable: true,
                speed: 1,
                opacity_min: 0.05,
            },
        },
    },
    retina_detect: true,
};
