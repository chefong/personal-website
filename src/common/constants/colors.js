export default {
    light: '#fffefb',
    dark: '#222222',
    tan: '#f1e7d0',
};
