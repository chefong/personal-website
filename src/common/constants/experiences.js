export default [
    {
        imagePath: {
            light: 'imgs/experience/opentable.png',
            dark: 'imgs/experience/opentable-white.png',
        },
        nameId: 'EXPERIENCE_OPENTABLE',
        positionId: 'SOFTWARE_ENGINEER_1',
        duration: 'February 2021 - Present',
        teamId: 'EXPERIENCE_OPENTABLE_TEAM_2',
        initiativesId: 'EXPERIENCE_OPENTABLE_INITIATIVES_2',
        technologiesId: 'EXPERIENCE_OPENTABLE_TECHNOLOGIES_2',
    },
    {
        imagePath: {
            light: '/imgs/experience/intuit.svg',
            dark: '/imgs/experience/intuit-white.svg',
        },
        nameId: 'EXPERIENCE_INTUIT',
        positionId: 'SOFTWARE_ENGINEER_1',
        duration: 'July 2020 - February 2021',
        teamId: 'EXPERIENCE_INTUIT_TEAM',
        initiativesId: 'EXPERIENCE_INTUIT_INITIATIVES',
        technologiesId: 'EXPERIENCE_INTUIT_TECHNOLOGIES',
    },
    {
        imagePath: {
            light: 'imgs/experience/opentable.png',
            dark: 'imgs/experience/opentable-white.png',
        },
        nameId: 'EXPERIENCE_OPENTABLE',
        positionId: 'SOFTWARE_ENGINEERING_INTERN',
        duration: 'June 2019 - August 2019',
        teamId: 'EXPERIENCE_OPENTABLE_TEAM_1',
        initiativesId: 'EXPERIENCE_OPENTABLE_INITIATIVES_1',
        technologiesId: 'EXPERIENCE_OPENTABLE_TECHNOLOGIES_1',
    },
];
