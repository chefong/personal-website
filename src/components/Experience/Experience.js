import styles from './Experience.module.css';

export default function Experience() {
  return (
    <section className={`${styles.section} container-fluid`}>
      <div className="row justify-content-center">
        <div className="col-md-8">
          <div className={styles.headerGroup}>
            <h2 className={styles.header}><span className={styles.headerNum}>02.</span> My Experience</h2>
          </div>
        </div>
      </div>
      <div className={styles.experienceGroup}>
        <div className="row justify-content-center">
          <div className="col-md-3">
            <img className={styles.companyImage} src="/imgs/experience/intuit.png" alt=""/>
          </div>
          <div className="col-md-5">
            <div className={styles.companyInfo}>
              <p className={styles.name}>Intuit</p>
              <p className={styles.position}>Software Engineer (July 2020 - Present)</p>
              <hr className={styles.line} />
              <p className={styles.description}>Financial Data Platform Team.</p>
            </div>
          </div>
        </div>
      </div>
      <div className={styles.experienceGroup}>
        <div className="row justify-content-center">
          <div className="col-md-3">
            <img className={styles.companyImage} src="/imgs/experience/opentable.png" alt=""/>
          </div>
          <div className="col-md-5">
            <div className={styles.companyInfo}>
              <p className={styles.name}>OpenTable</p>
              <p className={styles.position}>Software Engineering Intern (June 2019 - August 2019)</p>
              <hr className={styles.line} />
              <p className={styles.description}>As a part of the Content Team, I worked on various consumer-facing products (internal tools, Restaurant Profile, GuestCenter Admin) using React, TypeScript, Jest, and OpenComponents. One of the projects I worked on focused on improving an existing internal tool to speed up workflow for internal marketing users. Another project I worked on was kickstarting the Gallery Photos feature on OpenTable's GuestCenter Admin with my team, giving restaurants the ability to manage their own photos.</p>
            </div>
          </div>
        </div>
      </div>
    </section>
  )
}