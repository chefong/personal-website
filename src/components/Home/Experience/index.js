import { forwardRef, useContext } from 'react';
import { FormattedMessage } from 'text-provider';
import styles from './styles.module.scss';
import { store } from '~/store/GlobalProvider';

const experiences = [
    {
        imagePath: {
            light: 'imgs/experience/opentable.svg',
            dark: 'imgs/experience/opentable-white.svg',
        },
        nameId: 'EXPERIENCE_OPENTABLE',
        positionId: 'EXPERIENCE_OPENTABLE_ROLE',
        duration: 'February 2021 - Present',
        teamId: 'EXPERIENCE_OPENTABLE_TEAM',
        initiativeIds: [
            'EXPERIENCE_OPENTABLE_INITIATIVES_1',
            'EXPERIENCE_OPENTABLE_INITIATIVES_2',
            'EXPERIENCE_OPENTABLE_INITIATIVES_3',
        ],
        technologiesId: 'EXPERIENCE_OPENTABLE_TECHNOLOGIES_2',
    },
    {
        imagePath: {
            light: '/imgs/experience/intuit.svg',
            dark: '/imgs/experience/intuit-white.svg',
        },
        nameId: 'EXPERIENCE_INTUIT',
        positionId: 'EXPERIENCE_INTUIT_ROLE',
        duration: 'July 2020 - February 2021',
        teamId: 'EXPERIENCE_INTUIT_TEAM',
        initiativeIds: [
            'EXPERIENCE_INTUIT_INITIATIVES_1',
            'EXPERIENCE_INTUIT_INITIATIVES_2',
            'EXPERIENCE_INTUIT_INITIATIVES_3',
            'EXPERIENCE_INTUIT_INITIATIVES_4',
        ],
        technologiesId: 'EXPERIENCE_INTUIT_TECHNOLOGIES',
    },
    {
        imagePath: {
            light: 'imgs/experience/opentable.svg',
            dark: 'imgs/experience/opentable-white.svg',
        },
        nameId: 'EXPERIENCE_OPENTABLE',
        positionId: 'EXPERIENCE_OPENTABLE_INTERN_ROLE',
        duration: 'June 2019 - August 2019',
        teamId: 'EXPERIENCE_OPENTABLE_TEAM_INTERN',
        initiativeIds: [
            'EXPERIENCE_OPENTABLE_INITIATIVES_INTERN_1',
            'EXPERIENCE_OPENTABLE_INITIATIVES_INTERN_2',
        ],
        technologiesId: 'EXPERIENCE_OPENTABLE_TECHNOLOGIES_1',
    },
];

const Experience = (props, ref) => {
    const { state } = useContext(store);
    const { theme } = state;

    return (
        <section
            className={`${styles.section} container-fluid`}
            id="experience"
            data-testid="Experience-container"
            ref={ref}
        >
            <div className="row justify-content-center">
                <div className="col-lg-8 col-md-10 col-sm-10">
                    <div className={styles.headerGroup}>
                        <h2 className={styles.header}>
                            <span className={styles.headerNum}>03.</span> My
                            Experience
                        </h2>
                    </div>
                </div>
            </div>
            {experiences.map(
                ({
                    imagePath,
                    nameId,
                    positionId,
                    duration,
                    teamId,
                    initiativeIds,
                    technologiesId,
                }) => (
                    <div
                        className={styles.experienceGroup}
                        key={nameId}
                        data-testid="Experience-item"
                    >
                        <div className="row justify-content-center">
                            <div className="col-lg-3 col-md-4 col-sm-4">
                                <img
                                    className={styles.companyImage}
                                    src={
                                        theme === 'dark' && imagePath.dark
                                            ? imagePath.dark
                                            : imagePath.light
                                    }
                                    alt={nameId}
                                />
                            </div>
                            <div className="col-lg-5 col-md-6 col-sm-6">
                                <div className={styles.companyInfo}>
                                    <p className={styles.name}>
                                        <FormattedMessage id={nameId} />
                                    </p>
                                    <p className={styles.position}>
                                        <FormattedMessage id={positionId} /> (
                                        {duration})
                                    </p>
                                    <hr className={styles.line} />
                                    <p className={styles.description}>
                                        <FormattedMessage id={teamId} />
                                    </p>
                                    <ul className={styles.initiativesList}>
                                        {initiativeIds.map((initiativeId) => (
                                            <li
                                                key={initiativeId}
                                                className={styles.initiative}
                                            >
                                                <FormattedMessage
                                                    id={initiativeId}
                                                    values={{
                                                        linkClass: styles.link,
                                                    }}
                                                />
                                            </li>
                                        ))}
                                    </ul>
                                </div>
                            </div>
                        </div>
                    </div>
                ),
            )}
        </section>
    );
};

export default forwardRef(Experience);
