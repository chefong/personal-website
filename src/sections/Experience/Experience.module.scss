.section {
  margin: 10em auto;
}

.header {
  color: var(--theme-page-text);
  font-weight: bold;
}

.headerGroup {
  font-weight: bold;
  margin-bottom: 2.5em;
}

.headerNum {
  color: var(--theme-color);
}

.experienceGroup {
  margin-bottom: 4em;
}

.companyImage {
  display: block;
  margin: 0 auto;
  width: 75%;
  margin-bottom: 16px;
}

.companyInfo {
  color: var(--theme-page-text);
}

.name {
  font-weight: bold;
}

.position {
  font-style: italic;
  margin-top: -12px;
}

.line {
  width: 10%;
  border-width: 2px;
  border-color: #E5E5E5;
  margin-left: 0;
  margin-top: -8px;
}