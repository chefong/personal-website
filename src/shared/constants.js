const navItems = [
  {
    to: "about",
    num: "01.",
    name: "About"
  },
  {
    to: "projects",
    num: "02.",
    name: "Projects"
  },
  {
    to: "experience",
    num: "03.",
    name: "Experience"
  }
];

const projects = [
  {
    name: "Patch",
    description: "A web application that retroactively fixes your Zoom audio recordings.",
    githubLink: "https://github.com/willshiao/Patch",
    wonHackathon: true
  },
  {
    name: "CheaterBeater",
    description: "Identify and act on fraudulent Devpost hackathon submissions.",
    githubLink: "https://github.com/willshiao/CheaterBeater",
    wonHackathon: true
  },
  {
    name: "Baut",
    description: "A chatbot that recommends movies and music based on the user's likings.",
    githubLink: "https://github.com/KirbyDownB/GEICOChatBot",
    wonHackathon: true
  },
  {
    name: "birdie",
    description: "A web application that visualizes tweets on a Mapbox map.",
    githubLink: "https://github.com/chefong/birdie",
    wonHackathon: false
  },
  {
    name: "Stock Overflow",
    description: "Use news data to predict how a stock will act in the future.",
    githubLink: "https://github.com/kimjihwan0208/stockoverflow",
    wonHackathon: false
  },
  {
    name: "Joyride",
    description: "Determine the most efficient way to reach a destination with your carpool buddies!",
    githubLink: "https://github.com/ericong18/joyride",
    wonHackathon: true
  }
];

const experiences = [
  {
    imagePath: "/imgs/experience/intuit.png",
    name: "Intuit",
    position: "Software Engineer",
    duration: "July 2020 - Present",
    description: "Financial Data Platform Team."
  },
  {
    imagePath: "imgs/experience/opentable.png",
    name: "OpenTable",
    position: "Software Engineering Intern",
    duration: "June 2019 - August 2019",
    description: "As a part of the Content Team, I worked on various consumer-facing products (internal tools, Restaurant Profile, GuestCenter Admin) using React, TypeScript, Jest, and OpenComponents. One of the projects I worked on focused on improving an existing internal tool to speed up workflow for internal marketing users. Another project I worked on was kickstarting the Gallery Photos feature on OpenTable's GuestCenter Admin with my team, giving restaurants the ability to manage their own photos."
  }
];

const handles = [
  {
    name: "GitHub",
    iconPath: "/imgs/footer/github.svg",
    link: "https://github.com/chefong"
  },
  {
    name: "LinkedIn",
    iconPath: "/imgs/footer/linkedin.svg",
    link: "https://www.linkedin.com/in/ericong18/"
  },
  {
    name: "Instagram",
    iconPath: "/imgs/footer/instagram.svg",
    link: "https://www.instagram.com/chef_ong/"
  },
  {
    name: "Medium",
    iconPath: "/imgs/footer/medium.svg",
    link: "https://medium.com/@ericong18"
  },
  {
    name: "Email",
    iconPath: "/imgs/footer/envelope.svg",
    link: "mailto:ericong18@gmail.com"
  }
];

const revealProps = {
  duration: 500,
  delay: 400,
  distance: "1.25em",
  bottom: true
};

export { navItems, projects, experiences, handles, revealProps };