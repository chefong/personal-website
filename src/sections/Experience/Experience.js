import styles from './Experience.module.css';
import { Fade } from 'react-reveal';
import { FormattedMessage } from 'text-provider';
import { experiences, revealProps } from '../../shared/constants';

export default function Experience() {
  return (
    <section className={`${styles.section} container-fluid`} id="experience">
      <div className="row justify-content-center">
        <div className="col-lg-8 col-md-10 col-sm-10">
          <Fade {...revealProps}>
            <div className={styles.headerGroup}>
              <h2 className={styles.header}><span className={styles.headerNum}>03.</span> My Experience</h2>
            </div>
          </Fade>
        </div>
      </div>
      {experiences.map(({ imagePath, nameId, position, duration, descriptionId }, index) => (
        <div className={styles.experienceGroup} key={index}>
          <div className="row justify-content-center">
            <div className="col-lg-3 col-md-4 col-sm-4">
              <Fade {...revealProps}>
                <img className={styles.companyImage} src={imagePath} alt={nameId}/>
              </Fade>
            </div>
            <div className="col-lg-5 col-md-6 col-sm-6">
              <div className={styles.companyInfo}>
                <Fade {...revealProps}>
                  <p className={styles.name}>
                    <FormattedMessage id={nameId} />
                  </p>
                  <p className={styles.position}>{position} ({duration})</p>
                  <hr className={styles.line} />
                  <p className={styles.description}>
                    <FormattedMessage id={descriptionId} />
                  </p>
                </Fade>
              </div>
            </div>
          </div>
        </div>
      ))}
    </section>
  )
}