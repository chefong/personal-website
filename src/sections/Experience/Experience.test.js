import React from 'react';
import { render } from '@testing-library/react';
import Experience from './Experience';
import { experiences } from '../../shared/constants';

describe('Experience', () => {
  it('renders', () => {
    const { getByTestId } = render(<Experience />);
    expect(getByTestId('Experience-container')).toBeDefined();
  });

  it('matches snapshot', () => {
    const { container } = render(<Experience />);
    expect(container).toMatchSnapshot();
  });

  it('contains correct number of experiences', () => {
    const { getAllByTestId } = render(<Experience />);
    expect(getAllByTestId('Experience-item')).toHaveLength(experiences.length);
  });
});