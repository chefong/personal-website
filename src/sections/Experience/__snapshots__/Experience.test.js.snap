// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Experience matches snapshot 1`] = `
<div>
  <section
    class="undefined container-fluid"
    data-testid="Experience-container"
    id="experience"
  >
    <div
      class="row justify-content-center"
    >
      <div
        class="col-lg-8 col-md-10 col-sm-10"
      >
        <div
          class="react-reveal"
        >
          <h2>
            <span>
              03.
            </span>
             My Experience
          </h2>
        </div>
      </div>
    </div>
    <div
      data-testid="Experience-item"
    >
      <div
        class="row justify-content-center"
      >
        <div
          class="col-lg-3 col-md-4 col-sm-4"
        >
          <img
            alt="EXPERIENCE_INTUIT"
            class="react-reveal"
            src="/imgs/experience/intuit.png"
          />
        </div>
        <div
          class="col-lg-5 col-md-6 col-sm-6"
        >
          <div>
            <p
              class="react-reveal"
            >
              <span />
            </p>
            <p
              class="react-reveal"
            >
              Software Engineer
               (
              July 2020 - Present
              )
            </p>
            <hr
              class="react-reveal"
            />
            <p
              class="react-reveal"
            >
              <span />
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      data-testid="Experience-item"
    >
      <div
        class="row justify-content-center"
      >
        <div
          class="col-lg-3 col-md-4 col-sm-4"
        >
          <img
            alt="EXPERIENCE_OPENTABLE"
            class="react-reveal"
            src="imgs/experience/opentable.png"
          />
        </div>
        <div
          class="col-lg-5 col-md-6 col-sm-6"
        >
          <div>
            <p
              class="react-reveal"
            >
              <span />
            </p>
            <p
              class="react-reveal"
            >
              Software Engineering Intern
               (
              June 2019 - August 2019
              )
            </p>
            <hr
              class="react-reveal"
            />
            <p
              class="react-reveal"
            >
              <span />
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
